### PR TITLE
Revert "CCZ integrity check for locale ids"

### PR DESCRIPTION
**Pull request by sravfeyn**
Reverts dimagi/commcare-hq#24200

This PR got released on India and ICDS which introduced this bug https://dimagi-dev.atlassian.net/browse/ICDS-553 The issue is that the ICDS apps have custom XML containing some locale ids, which make it to suite xml file but are not present in app_strings.txt which results in the occurrence of this [validation error](https://github.com/dimagi/commcare-hq/pull/24200/files#diff-9cec308bf21cad9a25b9a3000d599426R241) from the original PR. Some examples
1. `m5.case_short.title` in custom_xml in [AWW app module 6](https://india.commcarehq.org/a/icds-cas/apps/view/62a3e97f4b8e5d11977d44160865121b/module/5f26c9862305536bc958e658cc3f5633e2a249eb/#case-detail-screen-config-tab) and [IT App module 6](https://india.commcarehq.org/a/icds-cas/apps/view/62a3e97f4b8e5d11977d441608650a17/module/e3dc238b663e86600937212196e768fcba2bd7b4/#case-detail-screen-config-tab).
2. `m9.case_short.title` in custom_xml in [AWW app module 10](https://india.commcarehq.org/a/icds-cas/apps/view/62a3e97f4b8e5d11977d44160865121b/module/82c984b1d3d603f4d83d3d3328d727a29a2a3f14/#case-detail-screen-config-tab)

I have checked with Marissa who mentioned that these custom xml elements have been there since around 2017 when Aliza added them. So, we don't have background on the intention of adding this custom XML anymore. It's a bit beyond interrupt to dig into why this PR adds a stricter validation which are breaking ICDS's custom XML and might result in similar edge cases in Production, so I am going to revert this PR for now for ICDS and let @orangejenny  take this up.

Urgent @mkangia @Rohit25negi

### CODE REVIEW

File: corehq/apps/hqmedia/tasks.py [hunk 1]
```diff
@@ -182,13 +181,24 @@ def build_application_zip(include_multimedia_files, include_index_files, app,
...
+                with zipfile.ZipFile(tmp, "r") as z:
+                    media_suites = [f for f in z.namelist() if re.search(r'\bmedia_suite.xml\b', f)]
+                    if len(media_suites) != 1:
+                        message = _('Could not identify media_suite.xml in CCZ')
+                        errors.append(message)
+                    else:
+                        with z.open(media_suites[0]) as media_suite:
+                            from corehq.apps.app_manager.xform import parse_xml
+                            parsed = parse_xml(media_suite.read())
+                            resources = {node.text for node in
```
Comment (Rohit25negi) [inline corehq/apps/hqmedia/tasks.py]: Why it is to be a set not just a list. Also We are sure the order does not matter here?

**Review by Rohit25negi — COMMENTED** [review]
Looks good Just left one question

File: corehq/apps/hqmedia/tasks.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/hqmedia/tasks.py]: Because, presumably, a set would avoid catching duplicate errors in the lines that follow.

**Review by Rohit25negi — APPROVED** [review]
looks good to me

**Comment by sravfeyn** [comment]
Merging this, even though the original PR fixes another issue, it's not as critical as the one this introduces.

**Pull request merged**